cores/esp32/esp32-hal-uart.c

```diff
@@ -497,6 +497,7 @@ uart_t *uartBegin(
     log_v("UART%d not installed. Starting installation", uart_nr);
   }
   uart_config_t uart_config;
+  uart_config.flags.backup_before_sleep = false; // necessary within IDF v5.3
```

Comment:
for future proofing I would suggest to memset the whole struct to 0. This should really be the case on every such occasion.